fixtures/cameo/nanospot-120.json

```diff
@@ -149,14 +149,16 @@
           "dmxRange": [16, 124],
           "type": "GoboIndex",
           "indexStart": 1,
-          "indexEnd": 6
+          "indexEnd": 6,
+          "comment": "Gobo 1 … Gobo 6"
```

Comment:
It could, of course, but changing everything again would be an effort that is soon overwritten anyway by the `wheels` branch. So I guess that the manual approach is good enough.